source/documentation/using_the_api/api_reference.md

```diff
@@ -1,22 +1,20 @@
 ## <a name="apireference">API reference</a>
 
-### <a name="getregister">`GET /register`</a>
+### <a name="get-register">View information about a register</a>
 
-View information about a register, including:  
+Endpoint: `GET /register/`
 
-* the internet domain the register is on
-* when the register was last updated
-* how many entries and records the register contains
-
-You can check the entry number you have locally is up to date with the register by comparing the value of `total-entries` for each. 
-
-Example URL: `https://local-authority-eng.register.gov.uk/register`
-
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/register/ --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
-
-Example response:
+```
```

Comment:
If you add the mark `http` any markdown processor like github's or any that relies on highlight.js will highlight this block accordingly.

E.g.

```md
```http
GET /regitser/ HTTP/2
Host: local-authority-eng.register.gov.uk
...
```
```